cmd/main.go

```diff
@@ -504,3 +513,51 @@ func uploadBaseISOWithLeader(uploadLeader leader.ElectorInterface, objectHandler
 		return objectHandler.UploadFile(ctx, Options.RHCOSBaseImage, s3wrapper.BaseObjectName)
 	})
 }
+
+func setupKubeAPI(log logrus.FieldLogger, enable bool) {
+	if !enable {
+		return
+	}
+	var schemes = runtime.NewScheme()
+	utilruntime.Must(scheme.AddToScheme(schemes))
+	utilruntime.Must(adiiov1alpha1.AddToScheme(schemes))
+
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
```

Comment:
creating manager here won't allows us to use its client ( and cache ) for bm inventory.

Reply:
You are correct, i did it like this for now because of: 
```
cyclomatic complexity 37 of func `main` is high (> 30) 
```
Will fix main.

Reply:
https://github.com/openshift/assisted-service/pull/830